src/index.js

```diff
@@ -65,6 +65,23 @@ class MetisMenu {
       .children(conf.triggerElement)
       .attr('aria-expanded', 'true'); // add attribute aria-expanded=true the triggers of all parents
 
+    $(this.element)
+      .find(`${conf.parentTrigger}:not(:has(${conf.subMenu}))`)
+      .children(conf.triggerElement)
+      .on(Event.CLICK_DATA_API, function () { // eslint-disable-line func-names
+        const eTar = $(this);
+        const parentTar = eTar.parents(conf.parentTrigger).last()[0];
+        if (eTar.parent(conf.parentTrigger).hasClass(ClassName.ACTIVE)) {
+          return;
+        }
+        eTar.parent(conf.parentTrigger).addClass(ClassName.ACTIVE);
+        eTar.parents(`${conf.subMenu}.${ClassName.METIS}`).children(`${conf.parentTrigger}.${ClassName.ACTIVE}:not(:has(${conf.subMenu}.${ClassName.SHOW}.${ClassName.COLLAPSED}:has(${conf.parentTrigger}.${ClassName.ACTIVE})))`).not(parentTar).children(conf.triggerElement)
+          .attr('aria-expanded', 'false');
+        eTar.parents(`${conf.subMenu}.${ClassName.METIS}`).children(`${conf.parentTrigger}.${ClassName.ACTIVE}:not(:has(${conf.subMenu}.${ClassName.SHOW}.${ClassName.COLLAPSED}:has(${conf.parentTrigger}.${ClassName.ACTIVE})))`).not(parentTar).children(conf.subMenu)
```

Comment:
Please, don't hide it manually, use "hide" function.

Reply:
Your change is nice. For animation and event listener to work properly, the "hide" function of metisMenu should be used. Like this:

```js
const notParentTar = eTar
.parents(`${conf.subMenu}.${ClassName.METIS}`)
.children(`${conf.parentTrigger}.${ClassName.ACTIVE}:not(:has(${conf.subMenu}.${ClassName.SHOW}.${ClassName.COLLAPSED}:has(${conf.parentTrigger}.${ClassName.ACTIVE})))`)
.not(parentTar);

notParentTar.children(conf.triggerElement).attr('aria-expanded', 'false');

self.hide(notParentTar.children(conf.subMenu)); // hiding subMenu
```